_layouts/default.html

```diff
@@ -7,7 +7,19 @@
     <link rel="stylesheet" href="/assets/css/styles.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
     <script src="https://code.jquery.com/jquery-3.3.1.js" integrity="sha256-2Kok7MbOyxpgUVvAk/HJ2jigOSYS2auK4Pfzbm7uH60=" crossorigin="anonymous"></script>
-    <script>
+      <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+      <script>
+          function removeTags(str) {
+              if ((str===null) || (str===''))
+                  return false;
+              else
+                  str = str.toString();
+
+              // Regular expression to identify HTML tags in
+              // the input string. Replacing the identified
+              // HTML tag with a null string.
+              return str.replace( /(<([^>]+)>)/ig, '');
```

Comment:
Super dangereux de virer les tags HTML comme ça, ici tu va virer tout ce qui est compris entre des chevrons, ça paraît trop large.

La bonne astuce en JS pour faire ça c'est de créé un élément HTML à la volé, et de récupérer le innerText

```suggestion
              const element = document.createElement("div");
              element.innerHTML = str;
              return element.innerText
```